### PR TITLE
v.2 paildizau automobiliu kiekis bei apskaitos duomenu kintamieji

### PR DESCRIPTION
**Pull request by AndDauJA**
papildziau automobilio duomenimis.

### CODE REVIEW

File: company-workers/src/main/java/CompanyParameters.java [hunk 1]
```diff
@@ -16,5 +18,18 @@ public static void comapnyInfo() {
...
+        int automob=5;
+        System.out.println("automobiliu kiekis imoneje:"+automob);
+
+    }
+    public static void kuroApskaita(){
+        int kurkoKiekis;
+        int kmKiekis;
+        String vairuotojas;
+
+
```
Comment (ernestaskriskus) [inline company-workers/src/main/java/CompanyParameters.java:33]: Nuostabus kodas

**Review by AntiQnis — APPROVED** [review]
tinka man ir sitas

File: company-workers/src/main/java/CompanyParameters.java [hunk 2]
```diff
@@ -16,5 +18,18 @@ public static void comapnyInfo() {
         workerNumber=136;
```
Comment (pavelvrublevskij) [inline company-workers/src/main/java/CompanyParameters.java:18]: I suppose here should be constant maybe?

File: company-workers/src/main/java/CompanyParameters.java [hunk 3]
```diff
@@ -1,3 +1,5 @@
+import javax.sound.midi.Soundbank;
+
 public class CompanyParameters {
```
Comment (pavelvrublevskij) [inline company-workers/src/main/java/CompanyParameters.java:5]: empty lines

**Review by pavelvrublevskij — CHANGES_REQUESTED** [review]
code smell

File: company-workers/src/main/java/CompanyParameters.java (hunk 3 above)
Comment (AndDauJA) [reply company-workers/src/main/java/CompanyParameters.java:5]: man grazu daug tusciu liniu :)

File: company-workers/src/main/java/CompanyParameters.java (hunk 2 above)
Comment (AndDauJA) [reply company-workers/src/main/java/CompanyParameters.java:18]: could be, could be but now is the best update.

**Pull request merged**